Formula/mpv.rb

```diff
@@ -85,6 +85,9 @@ def install
     system "python3", "waf", "install"
 
     if build.with? "bundle"
+      # allow the build to work on older versions of OS X
+      # see https://github.com/mpv-player/mpv/commit/c176f1169f356059ee6e3c447e4546fcdb335ec2
+      ENV["MACOSX_DEPLOYMENT_TARGET"] = "10.3"
```

Comment:
```
ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
```
is safer. This needs to be reported upstream, though, because it shouldn't be necessary for us to tweak this is in the environment to get a successful build.

Reply:
Thanks. Reported.